www/_posts/2021-06-28-whitelist-plugin-release.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+author:
+    name: Pieter Van Poyer
+title:  "Whitelist Plugin Released!"
+categories: news
+tags: release plugins
+---
+
+We are happy to announce that we have just released an update to our Whitelist plugin!
+
+* [cordova-plugin-whitelist@1.3.5](https://github.com/apache/cordova-plugin-whitelist)
+
+## Release Highlights
+
+### Network Information

```

Comment:
"Network Information" seems like maybe a copy/paste error? I think it can be removed